S100-Permit-Service-Open-Api-Sepc-v0.1.yaml

```
@@ -0,0 +1,162 @@
+openapi: 3.0.1
+info:
+  title: UKHO S100 Permit Service API
+  description: |
+    The UKHO S100 Permit Service API.
+  contact:
+    name: UKHO
+    email: contact@ukho.co.uk
+  version: 0.0.1
+  
+externalDocs:
+  url: https://github.com/UKHO/s-100-permit-service/tree/main
+servers:
+  - url: https://permit-service-url
+    description: Proposed URL
+
+security:
+  - jwtBearerAuth: []
+  
+paths:
+  /permits/{licenceId}:
+    get:
+      description: |
+        This service uses the S-100 Part 15 data protection scheme to generate unsigned PERMIT.XML files for the respective User Permit Numbers (UPNs) for the requested licence and returns the compressed zip file containing all these PERMIT.XML files.
+      operationId: getS100Permits
+      tags:
+        -  "UKHO Only"
+      parameters:
+      - name: licenceId
+        in: path
+        description: Licence Id. It must be an integer value and greater than zero.
+        required: true
+        schema:
+          type: integer
+          example: 12345678
+      responses:
+        "200":
+          description: OK - Returns permit files.
+          headers:
+            X-Correlation-ID:
+              description: GUID for the request for logging/tracing
+              schema:
+                type: string
+            Content-Disposition:
+              schema:
+                type: string
+                format: filename=permits.zip; filename*=UTF-8''Permits.zip
+          content:
+            application/zip:
+              schema:
+                title: S100 Permits ZIP File
+                type: string
+                format: binary
+        "204":
+          description: No Content - There are no S100 permits for the licence (No UPNs or Holdings found for the requested licence from S100 Shop Facade).
```

```suggestion
          description: No Content - There are no S100 permits for the licence.
```
I suggest internal wokings of API shouldn't be included.

---

Agree, Updated.